circle.yml

```diff
@@ -20,6 +25,52 @@ jobs:
           command: npm test -- --reporters=default --reporters=jest-junit --ci --runInBand
       - store_test_results:
           path: .
+
+  build-frontend:
+    docker:
+      - image: circleci/node:lts
+    working_directory: ~/project/frontend
+    steps:
+      - checkout:
+          path: ~/project
+      - restore_cache:
+          key: dependencies-{{ checksum "package-lock.json" }}
+      - run:
+          name: Installing frontend dependencies
+          command: npm install
+      - save_cache:
+          paths:
+            - node_modules
+          key: dependencies-{{ checksum "package-lock.json" }}
+      - run:
+          name: Build React frontend
+          command: npm run build
+      - persist_to_workspace:
+          root: build
+          paths:
+            - .
+
+  deploy-frontend-stage:
+    docker:
+      - image: circleci/python:3.8
+    steps:
+      - attach_workspace:
+          at: build-frontend
+      - aws-s3/sync:
+          from: build-frontend/
+          to: "s3://$FRONTEND_STAGE_AWS_S3_BUCKET/frontend"
+          overwrite: true
+          arguments: |
+            --acl public-read \
+            --profile=default
+          aws-access-key-id: FRONTEND_STAGE_AWS_ACCESS_KEY_ID
+          aws-secret-access-key: FRONTEND_STAGE_AWS_SECRET_ACCESS_KEY
+          aws-region: FRONTEND_STAGE_AWS_REGION
+      - run:
+          name: Invalidate CloudFront cache
+          command: |
+            aws cloudfront create-invalidation --distribution-id $FRONTEND_STAGE_AWS_CLOUDFRONT_DISTRIBUTION_ID --paths '/*' --profile=default
```

Comment:
Yeah, its required because otherwise the `awscli` orb picks up the credentials from the environment variables (that are the ones for the integration environment).

Reply:
What I meant is that we already have it defined [here](https://github.com/open-craft/opencraft/pull/494/files#diff-29944324a3cbf9f4bd0162dfe3975d88R91). Should it still be in the [current line](https://github.com/open-craft/opencraft/pull/494/files#diff-29944324a3cbf9f4bd0162dfe3975d88R98)? Sorry for not making this clear.

Reply:
Oh yeah, the same bug happens here too. If this argument isn't passed, they incorrect credentials are used. I think that this happens because `awscli` prioritizes environment variables over the `~/.aws/credentials` file, and the only way to circumvent this and leave the variables for the integration tests untouched is to pass the argument on every line run.

Reply:
Removed these workarounds in favor of using contexts